docs/paper/dev/api/commands.mdx

```diff
@@ -0,0 +1,106 @@
+---
+slug: /dev/commands
+description: A guide to Paper's Brigadier command API.
+---
+
+# Command API
+
+Paper's command system is built on top of Minecraft's Brigadier command system. This system is a
+powerful and flexible way to define commands and arguments.
+
+:::danger[Experimental]
+
+Paper's command system is still experimental and may change in the future.
+
+:::
+
+## Defining commands
+
+:::note
+
+This uses the <Javadoc name={"io.papermc.paper.plugin.lifecycle.event.LifecycleEventManager"}>LifecycleEventManager</Javadoc>
+to register the command. See the [Lifecycle Events](./lifecycle.mdx) page for more information.
+
+:::
+
+### JavaPlugin
+Commands can be registered inside the <Javadoc name={"org.bukkit.plugin.java.JavaPlugin#onEnable()"}>onEnable</Javadoc>
+method of the plugin. Commands registered here will not be available to datapack functions because
+functions are loaded by the server before plugins are loaded. To make commands available to
+datapacks, register them via the
+<Javadoc name={"io.papermc.paper.plugin.bootstrap.PluginBootstrap#bootstrap(io.papermc.paper.plugin.bootstrap.BootstrapContext)"}>PluginBootstrap</Javadoc>.
```

Comment:
should this just link the the plugin bootstrap section below, and then the section below should link to the paper plugin page